src/hooks/constants/lib/fetchConstants.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiPromise } from '@polkadot/api';
+import { Constants, Fee } from '../../../generated/graphql';
+
+export const repayFeeDataType = '(u32, u32)';
+
+export const unwrapRepayFeeToFee = (apiInstance: ApiPromise): Fee => {
+  const apiRepayFee = apiInstance
+    .createType(repayFeeDataType, apiInstance.consts.lbp.getRepayFee)
+    .toHuman() as Array<string>;
+
+  return {
+    numerator: apiRepayFee[0],
+    denominator: apiRepayFee[1],
+  };
+};
+export const fetchConstants = (apiInstance: ApiPromise): Constants => ({
+  lbp: {
+    repayFee: unwrapRepayFeeToFee(apiInstance),
+  },
+});
```

Comment:
- This should be getRepayFee and should live in its own file
- lbp constants should be a sub resolver for the abstract constants entity, that does not have a resolver of its own. This will allow for constants composability.
- the constants resolver should be modular, you should not couple lbp specific constants with the overall constants resolver, please separate it

I think these separate comments ☝️  are closely-coupled and the analysis and solution can be talked about in one thread ?  plz correct if that is wrong.

Analysis:  the `constants` resolver as I wrote it is not composable.  It needs to be modularized so that we can compose future queries for constants that only return the things needed in those hypothetical future contexts.

proposed Solution questions/Todos:
- is this a situation where there is a '[resolver chain](https://www.apollographql.com/docs/apollo-server/data/resolvers/#resolver-chains)' <-- do those docs describe the situation and solution ?

Reply:
Yeah resolver chains looks like what we did e.g. with balances, nested resolvers.

Reply:
tl;dr; point here is that if you want to fetch just lbp constants, the subresolver for lbp constants will do just that if your query asks for it. it wont go and fetch also xyk constants and others in one bulky resolver.

Reply:
@maht0rz plz review latest to see how I separated the concerns